include/clap/preset-discovery.h

```diff
@@ -136,8 +136,10 @@ typedef struct clap_preset_location {
    // name of this location
    char name[CLAP_NAME_SIZE];
 
-   // path to a directory or a file on the file system in which preset can be found
-   char path[CLAP_URI_SIZE];
+   // URI
+   // - file:/// for pointing to a file or directory; directories are scanned recursively
+   // - plugin://<clap-plugin-id> for presets which are bundled inside the plugin itself
```

Comment:
Using the plugin ID here is a great idea! For the file URIs it may be a good idea to add an example of surch a URI on Windows and on other platforms, just to make sure hosts and plugins are on the same page.